### PR TITLE
feat: update packages

### PR DESCRIPTION
**Pull request by agatha197**
This pull request updates a wide range of dependencies in the `package.json` and `pnpm-lock.yaml` files, ensuring the project remains up-to-date with the latest features and security patches. Changes include various updates to `@babel`, `@vaadin`, `@rollup`, and other dependencies. The updates range from minor version bumps to address bug fixes and performance improvements to more significant version upgrades that introduce new features and improvements.

---

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Review by yomybaby — CHANGES_REQUESTED** [review]
Please also update `/react/package.json`.

**Comment by agatha197** [comment]
> Please also update `/react/package.json`.

Done!

File: react/src/components/lablupTalkativotUI/LLMChatCard.tsx [hunk 1]
```diff
@@ -333,7 +334,10 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
           closable
         />
       ) : null}
-      <VirtualChatMessageList messages={messages} isStreaming={isLoading} />
+      <VirtualChatMessageList
+        messages={messages as Array<Message>}
```
Comment (yomybaby) [inline react/src/components/lablupTalkativotUI/LLMChatCard.tsx]: You don't neet to use `as`. I'll push a commit to remove `as`

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**